package.json

```diff
@@ -34,6 +34,7 @@
     "start:storybook:test": "NODE_ENV=test start-storybook --port 57021 --quiet --ci",
     "start:storybook:dev": "NODE_ENV=development  start-storybook --port 80",
     "dev": "concurrently --raw \"yarn start:storybook:dev\" \"yarn watch\"",
+    "test:unit": "vitest",
     "test": "cypress run",
```

Comment:
Consider changing the script for running Cypress to `test:integration`.

This would match the pattern that we use for other types of task. We might even then create a "top-level" `test` script which runs both types of tests but I am not sure whether it would be used.

Reply:
Makes sense. I'll change the command.